app/src/main/java/eu/kanade/tachiyomi/Migrations.kt

```diff
@@ -265,6 +265,12 @@ object Migrations {
                     preferences.navigationModePager().set(5)
                     preferences.navigationModeWebtoon().set(5)
                 }
+                if (prefs.getBoolean("display_number_of_category_items", false)) {
+                    preferences.categoryNumberOfItems().set(false)
+                }
+                if (prefs.getBoolean("display_number_of_library_items", false)) {
+                    preferences.libraryNumberOfItems().set(false)
+                }
```

Comment:
```suggestion
                val displayNumberOfItems = flowPrefs.getBoolean("display_number_of_items", false)
                if (displayNumberOfItems)
                    preferences.categoryNumberOfItems().set(false)
                }               
```
Also move this another if block after bumping app version

Reply:
Are you sure that ```display_number_of_items``` works?
Or did you mean ```display_number_of_category_items```?
In ```PreferencesHelper.kt``` the preference ```display_number_of_items``` was renamed to ```display_number_of_category_items```.

Reply:
No need to rename the preference if we don't introduce a new one.

Reply:
With this pr ```display_number_of_library_items``` is new.

Reply:
Right, but I suggested not adding a new setting.